server.js

```diff
@@ -150,6 +151,12 @@ async function work(body) {
     return;
   }
 
+  if(skipTitle == "WIP"){
```

Comment:
This logic is not correct. What you want to do is to skip if the title contains WIP. In this case you're going to skip everything if you set the config to WIP